src/core/client/admin/routes/moderate/components/ModerateCard.tsx

```diff
@@ -0,0 +1,126 @@
+import cn from "classnames";
+import { Localized } from "fluent-react/compat";
+import React, { StatelessComponent } from "react";
+
+import { PropTypesOf } from "talk-framework/types";
+import { Button, Card, Flex, Icon } from "talk-ui/components";
+
+import MarkersContainer from "../containers/MarkersContainer";
+import AcceptButton from "./AcceptButton";
+import CommentContent from "./CommentContent";
+import InReplyTo from "./InReplyTo";
+import styles from "./ModerateCard.css";
+import RejectButton from "./RejectButton";
+import Timestamp from "./Timestamp";
+import Username from "./Username";
+
+interface Props {
+  id: string;
+  username: string;
+  createdAt: string;
+  body: string;
+  inReplyTo: string | null;
+  comment: PropTypesOf<typeof MarkersContainer>["comment"];
+  status: "accepted" | "rejected" | "undecided";
```

Comment:
Is this something we can also link to the graphql typescript enum instead?

Reply:
This one is a presentational component and just defines what it accepts, regardless of what's in the schema.